src/argilla/client/feedback/dataset/local/dataset.py

```diff
@@ -63,7 +64,9 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-class FeedbackDataset(ArgillaMixin, HuggingFaceDatasetMixin, FeedbackDatasetBase[FeedbackRecord], TaskTemplateMixin):
+class FeedbackDataset(
+    ArgillaMixin, HuggingFaceDatasetMixin, FeedbackDatasetBase[FeedbackRecord], TaskTemplateMixin, MetricsMixin
```

Comment:
can you also create a unification mixin?

Reply:
Done!